src/pages/sustainability.js

```diff
@@ -20,13 +20,13 @@ const SustainabilityPage = () => {
       </div>
       <section className='section--sustainability-pledge'>
         <div className='inner'>
-          <h2>2024 Sustainability Pledge</h2>
+          <h2 id="Pledge">2024 Sustainability Pledge</h2>
```

Comment:
No objection to adding these, but CSS id and class names are conventionally lowercased